spec/support/shared_examples/models/changer_of_the_exemption_status.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
```

Comment:
Should this file be renamed to `activator_of_exemptions.rb`?

Reply:
Yes it should.

Reply:
This is fixed in 024adc8.